eng/AddDrtsToPayload.targets

```diff
@@ -0,0 +1,26 @@
+<Project>
+  <PropertyGroup>
+    <!-- AnyCPU/x86 builds don't produce an output folder in artifacts. This only exists for x64 builds. -->
+    <PlatformFolder Condition="'$(Platform)'=='x64'">x64</PlatformFolder>
+
+    <!-- For the test payload, we default to x86 for AnyCPU platform --> 
+    <PayloadPlatformFolder>x86</PayloadPlatformFolder>
+    <PayloadPlatformFolder Condition="'$(Platform)'=='x64'">x64</PayloadPlatformFolder>
+    <PayloadBaseLocation>$(RepoRoot)artifacts\test\$(Configuration)\$(PayloadPlatformFolder)</PayloadBaseLocation>
+    <PayloadDrtLocation>$(PayloadBaseLocation)\Test\DRT</PayloadDrtLocation>
+  </PropertyGroup>
+
+  <ItemGroup>
```

Comment:
As I said below, it feels like this ItemGroup should live in a props file under eng, where the more targetish bits can go down into the SDK.

Reply:
Reuse `$(Architecture)` instead of computing yet another property derived from `$(Platform)`

Reply:
Are we anticipating packaging the DRTs that built out in the open along with the infrastructure? I was thinking we'd just package the infrastructure and run a different suite of tests internal.

Reply:
This one is a bit confusing, and maybe i should add some more documentation on it, but Arcade doesn't provide the same set of properties to the `AfterSolutionBuild.proj` build that they do to the restore or build projects:

https://github.com/dotnet/arcade/blob/b5016f5688dc8ca9f3e4811ee7e2e86ad8907a40/src/Microsoft.DotNet.Arcade.Sdk/tools/Build.proj#L225